### ISSUE TITLE
"v0.2.1" With memory 8GB, after 90000block, something goes wrong and it does not synchronize.

### ISSUE THREAD

**Issue by dusanjp** [nemtech/symbol-bootstrap#62]
"v0.2.1"
With memory 8GB, after 90000block, something goes wrong and it does not synchronize.

cpu v2 / v6 memory 8GB All 3 servers did not synchronize
The node on the 16GB / 24GB / 32GB server successfully synchronized with both "roles: 3 / roles: 7".
This also occurred when "v0.2.0" was started later.
Isn't it a problem that has existed since "v0.2"? It seems that it became apparent for the first time after reaching 90000 blocks.

This time, it seems that there is a problem of synchronization / non-synchronization due to the factor of memory capacity, not the processing capacity of cpu.

**Comment by fboucquez** [comment]
Hi @dusanjp, 

We are revisiting the system requirements for peer+api nodes. We will update this issue and symbol docs when we have the new requirements.

**Comment by fboucquez** [comment]
@segfaultxavi do we have a guide for the requirements?

**Comment by segfaultxavi** [comment]
No, we don't. I'm still waiting for the actual list of requirements.

**Comment by fboucquez** [comment]
Doc has been fixed
